x/upgrade/module.go

```diff
@@ -171,8 +171,7 @@ func init() {
 	)
 }
 
-//nolint:revive
-type UpgradeInputs struct {
+type UpgradeInputs struct { //nolint:revive // for readability, we ignore var-naming rules here
```

Comment:
another pr moved this to ModuleInputs and Outputs can we make the same change here

Reply:
+1 given that there is little changes here too.
Possibly we could rename them for all modules in this PR as well, so we are sure to get the SDK consistent.